apps/db_assembler/db_assembler.sh

```diff
@@ -93,20 +93,22 @@ assembler()
 
 clear
 echo ""
-echo -e "\e[1;37m"
-echo "         Remember that the files are generated in the env/dist/sql folder."
-echo "         This tool is an alternative for those who prefer to execute SQL scripts manually."
+echo -e "\e[1;33m"
+echo "   We do not recommend using this script."
```

Comment:
I think it would be better to state that it's not the recommended method, this way it sounds like the tool just accepts that it's bad.
Instead of `We do not recommend using this script.`, I'd suggest: `This is not the recommended update method.`

Reply:
I accommodate it. Thanks for the suggestion.

Reply:
![message](https://user-images.githubusercontent.com/2810187/213956360-e0c306f6-f56a-485f-b7ba-2c1e65252df7.png)